internals/search.py

```diff
@@ -211,7 +214,13 @@ def process_query(
       logging.info('combining result so far with %r', feature_ids)
       result_id_set.intersection_update(feature_ids)
   result_id_list = list(result_id_set or [])
-  total_count = len(result_id_list)
+
+  # TODO(danielrsmith): Update this to more efficiently count the
```

Comment:
I think that you have hit upon a defect in my existing code for paginating with unlisted features: search results do not show unlisted features that the signed in user could edit.

However, I don't think that this is the right fix because retrieving the entire list of features before pagination means that we are retrieving a lot more features.  One of the main goals of this API is to be faster than our old approach by avoiding doing that.

So, I think that you could back out your changes to this file and just leave a TODO for now.   E.g., # TODO(jrobbins): results should include unlisted features that the signed in user can edit.

Reply:
I agree that this is not the right fix here (hence the TODO 😅). Maybe I can take a deeper look and implement a fix here that satisfies the efficiency needs and displays the correct number.